public/docs/_examples/tutorial/dart/pubspec.yaml

```diff
@@ -0,0 +1,15 @@
+name: angular2_tour_of_heroes
+version: 0.0.1
+environment:
+  sdk: '>=1.13.0 <2.0.0'
+dependencies:
+  angular2: 2.0.0-beta.1
+  browser: ^0.10.0
+  dart_to_js_script_rewriter: ^0.1.0+4
```

Comment:
@kevmoo It doesn't break dartium for development:

> This transformer only runs when pub is running in release mode.
https://github.com/Workiva/dart_to_js_script_rewriter

So this transformer does does nothing in development. I think it actually fixes dartium in *release* mode, because otherwise `browser/dart.js` will point dartium to dart files, that don't exists in release mode.